test/perf/hc-nested-list_perf.html

```diff
@@ -82,6 +82,14 @@
             it('should fetch a deeply nested items depth in <1ms', () =>
               expect(getPerformance(() =>
               element._getItemDepth(gChild || child || parent, parts)).avg).at.most(1));
+            it('should collapse open tree nodes in <1 ms', () => {
+              element.parts = parts;
+              Object.keys(element.parts).forEach(k => {
```

Comment:
Again, this has to run before every test, not just before the group of tests, since the test changes this data.  You'll have to add it as the second function call to `getPerformance`